index.html

```diff
@@ -0,0 +1,174 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <div id="aside1">
+      <nav id="navbar">
+        <header id="JS_Documentation"><h1>JS Documentation</h1>></header>
+          <a href="#Introduction" class="nav-link"><h1>Introduction</h1><br></a>
+          <a href="#What_you_should_already_know" class="nav-link"><h1>What you should already know</h1><br></a>
+          <a href="#JavaScript_&_Java" class="nav-link"><h1>JavaScript & Java</h1><br></a>
+          <a href="#Hello_world"class="nav-link"><h1>Hello world</h1><br></a>
+          <a href="#variables" class="nav-link"><h1>variables</h1><br></a>
+          <a href="#Declaring_variables" class="nav-link"><h1>Declaring variables</h1><br></a>
+          <a href="#Variable_scope" class="nav-link"><h1>Variable scope</h1><br></a>
+          <a href="#Global_variables" class="nav-link"><h1>Global variables</h1><br></a>
+      </nav>
+    </div>
+    <div>
+      <main id="main-doc">
+      <section id="Introduction" class="main-section">
+        <header ><h1>Introduction</h1></header>
+        <p>JavaScript is a cross-platform, object-oriented scripting language. It is a small and lightweight language. Inside a host environment (for example, a web browser), JavaScript can be connected to the objects of its environment to provide programmatic control over them.</p>
+        <p>JavaScript contains a standard library of objects, such as Array, Date, and Math, and a core set of language elements such as operators, control structures, and statements. Core JavaScript can be extended for a variety of purposes by supplementing it with additional objects; for example: </p>
```

Comment:
It is not considered best practice to right more than 100 characters in the same line. Please fix this.